runtime/onert/core/src/compiler/ExecutorFactory.cc

```diff
@@ -149,32 +150,17 @@ void ExecutorFactory::runTensorRegistration(ir::LoweredGraph *lowered_graph,
     else
     {
       // Default registration
-      for (const auto op_idx : op_seq)
+      for (auto index : contexts.at(backend)->operand_list())
       {
-        const auto &op = lowered_graph->graph().operations().at(op_idx);
-        for (const auto &index : (op.getInputs() | ir::Remove::UNDEFINED) + op.getOutputs())
-        {
-          if (!tensor_builder->isRegistered(index))
-          {
-            const auto &operand_lower_info =
-                lowered_graph->getLowerInfo(index)->def_factors().getOnlyElement();
-
-            // E.g., permute (CPU) -> tensor A -> MaxPool2D(acl_cl)
-            // op.getOutputs() of permute (CPU) returns tensor A
-            // but tensor A belongs to the backend of acl_cl.
-            // So, we have to make this tensor NOT registered for CPU.
-            if (operand_lower_info.backend() != backend)
-              continue;
```

Comment:
Now we are now based on `BackendContext::operand_list`, this case does not occur.